### DISCUSSION TITLE
獨家》中研院攜手國網中心打造AI模型 可預測規模6以上地震

### DISCUSSION THREAD

**Discussion by Kotori0501** [1101-GS4719-C/Python-Community-GS4719-C#830] [General]
如果未來電離層全電子含量（TEC）能跟AI結合，會使預測地震的準確度提高，也可以預測大規模地震，或許可以能在地震前發佈預警，給予時間避難。
https://news.ltn.com.tw/news/life/breakingnews/4105892

**Comment by yuxinlin93** [comment]
希望這個技術可以趕快應用在政府的地震警報，救救我這個對地震毫無感知的人類

**Comment by dabbie109303501** [comment]
希望能趕快成功研發出來，這樣就能減少發生大地震時的傷亡數了

**Comment by Pietra1226** [comment]
希望可以趕快應用到現實社會，最近地震又開始頻繁了。每次地震發生再開始思考要不要逃跑好像太慢了。

**Comment by HuangHsinTzu** [comment]
太棒了！地震真的太可怕了
大地震常常導致家破人亡
那些災民往後的人生可能都因為大地震整個都毀了
如果可以實現預測地震並配合有效率的疏散
一定可以避免很多悲劇的發生！

**Comment by ghost** [comment]
這次TEC結合ai來確實對預測地震有很大的進展，但由於無法確定TEC與地震發生是若且若為的關係，所以準確性依就不高，若未來能發現更多與地震相關的參數，預測地震或許就不在是夢想。

**Comment by Twenty0522** [comment]
地震前看到新聞的預報感覺超酷的，有種預知未來的感覺。相信未來對於地震的預測可以減少許多傷害，而現今對於小地震的預報也可以提醒年長者及幼童或其照顧者，在這期間注意年長者如廁、幼童上下樓梯或是其他日常起居。

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
希望這項技術能盡早發展完全 讓人類有更多的時間來處理面臨的問題

**Comment by StevenWanglolz** [comment]
希望這項技術能盡早完成，這樣可以造福好多好多人，除了日本和台灣以外，許多國家也都受地震所害

**Comment by Evan0979** [comment]
希望以後越來越多的天災都能夠被預測,不又再有許多無辜的人民因此受難

**Comment by wenchi9023** [comment]
希望這項技術能夠越來越健全，以減少更多地震帶來的傷亡。但台灣時不時就來地震，大多數人也習以為常，極少數人才會在地震來臨時真的做保護的動作，因此與其預測地震何時來臨，我認為不如平時就要做好準備，定期加強避難教育及疏散路線等才是最好保護你我的方法。

**Comment by chia-kai-chang** [comment]
test

**Comment by chia-kai-chang** [comment]
希望這項技術能夠越來越健全，以減少更多地震帶來的傷亡。但台灣時不時就來地震，大多數人也習以為常，極少數人才會在地震來臨時真的做保護的動作，因此與其預測地震何時來臨，我認為不如平時就要做好準備，定期加強避難教育及疏散路線等才是最好保護你我的方法。

**Reply by chia-kai-chang** [reply]
test

**Comment by chia-kai-chang** [comment]
test2

**Comment by chia-kai-chang** [comment]
test2

**Reply by chia-kai-chang** [reply]
111

**Comment by chia-kai-chang** [comment]
我是廢物大叔

**Reply by chia-kai-chang** [reply]
然後AI還說我是廢文 我哭~

**Reply by chia-kai-chang** [reply]
123

**Reply by chia-kai-chang** [reply]
123456

**Comment by peggy0329** [comment]
我是一個對地震極度敏感的人
每次都會被地震嚇死 好需要這種預知的系統
這個系統結合了地科、物理及化學等領域的知識來分析地震預知問題
而我想到有些動物對於地震即將來臨也有會有事先的察覺
科學家們是否也可以透過多一項結合生物的雷達來增進預知地震的能力？

**Comment by Zoe06060606** [comment]
這半年的地震有夠頻繁 真的希望這技術能早日成熟 搭配預警系統 以及通知警消救護車 相信這樣能避免很多悲憾

**Comment by Doreen331** [comment]
昨天才剛看到有人爬玉山爬到一半結果遇到大地震，幸好沒人去世，但還是很可怕，如果這個技術早日成熟應該可以減少像這種事件的傷害

**Comment by Floriaand** [comment]
我覺得重要的是發布警報的時間，提前至少半小時才有幫助，如果只能提前幾分鐘，連遺書都寫不完。

**Comment by WEI-CHEN-CHUN** [comment]
地震雖尚無法完全預測，若此項技術可應用，會帶來很大幫助

**Comment by ru0372572** [comment]
原來地底下的板塊運動會跟在天空中那麼遙遠的電離層有關係呀！地球物理還是有很多我們不懂的變因在影響，希望研究能一步步接近完整，除了讓我們更了解這個世界運作原理外，還能有足夠時間防範大地震。
